packages/history-service/src/__tests__/create-root-location-transformer.test.ts

```diff
@@ -242,4 +242,41 @@ describe('#createRootLocationTransformer', () => {
       });
     });
   });
+  describe('#createLocation', () => {
+    describe('without a primary', () => {
+      it('joins all consumer locations together as a single encoded query param ', () => {
```

Comment:
`joins all non-primary consumer locations together as a single encoded query param` and please remove the last space

Reply:
done